src/auto_archiver/archivers/twitter_archiver.py

```diff
@@ -52,11 +53,21 @@ def download(self, item: Metadata) -> Metadata:
         result = Metadata()
 
         scr = TwitterTweetScraper(tweet_id)
+        do_alternative = False
```

Comment:
What is the goal of this hardcoded value? It will never trigger the condition below.